source/geometries/Next100Shielding.cc

```diff
@@ -217,74 +311,135 @@ namespace nexus {
       steel_box_logic->SetVisAttributes(grey_col);
       G4VisAttributes antiox_col = nexus::BloodRed();
       //  antiox.SetForceSolid(true);
-      roof_logic->SetVisAttributes(antiox_col);
+      roof_logic    ->SetVisAttributes(antiox_col);
       lat_beam_logic->SetVisAttributes(antiox_col);
-      struct_logic->SetVisAttributes(antiox_col);
+      struct_logic  ->SetVisAttributes(antiox_col);
```

Comment:
`front_beam_logic` should be also added here: in the current version of the code, the front beams are visible even when `visibility == 0`.